contrib_bots/lib/giphy/giphy.py

```diff
@@ -0,0 +1,140 @@
+# To use this plugin, you need to set up the Giphy API key for this bot in
+# ~/.giphy_config
+
+from __future__ import absolute_import
+from __future__ import print_function
+import six
+import requests
+import logging
+import sys
+import os
+import re
+if six.PY2:
```

Comment:
`from six.moves.configparser import SafeConfigParser` should work for either version here.